.github/workflows/pytorch-version-tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.6, 3.7, 3.8]
-        pytorch-version: [1.7.0, 1.6.0, 1.5.1, 1.4.0, 1.3.1]
+        pytorch-version: [1.7.1, 1.6.0, 1.5.1, 1.4.0, 1.3.1]
```

Comment:
Maybe add 1.8.0?

Reply:
Usually, we add only previous versions here as the latest stable one is running in unit-tests.yml...